src/car/distance/DistanceCar.hpp

```diff
@@ -110,7 +110,7 @@ class DistanceCar : virtual public SimpleCar
      * car.setSpeed(speedInPercentageOfMotorPower);
      * \endcode
      */
-    void setSpeed(float speed) override;
+    virtual void setSpeed(float speed) override;
```

Comment:
:warning: **modernize-use-override** :warning:
'virtual' is redundant since the function is already declared 'override'
```suggestion
    virtual void ```